examples/CMakeLists.txt

```diff
@@ -28,7 +28,6 @@ if (APPLE AND NOT CMAKE_SYSTEM STRLESS "Darwin-18.0.0")
   add_definitions(-DGL_SILENCE_DEPRECATION)
   MESSAGE(AUTHOR_WARNING "OpenGL is deprecated starting with macOS 10.14 (Mojave)!")
 endif()
-list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/rlgl_standalone.c)
```

Comment:
Why can this be removed? Does rlgl_standalone no longer clash with raylib proper? The commit message should explain that.

Reply:
Yes, they no longer crash, the examples build correctly on my machine. @a3f